src/environment/environment.local.ts

```diff
@@ -18,5 +18,6 @@ export const environment: LocalEnvironmentFile = {
     journalUrl: '/assets/mock/local-journal.json',
     autoRefreshInterval: 20000,
     numberOfDaysToView: 7,
-  }
+  },
+  loggingUrl: 'https://calvin.mes.dev-dvsacloud.uk/v1/logs/',
```

Comment:
We sticking with Calvin's env for now?

Reply:
I have removed the url as we shouldn't be using calvin's and it wouldn't work when authentication is turned on. We need a way to run the logging locally but that can be hashed out in the ticket to implement logging on the app.